fastlane/helpers/android_emulator_helper.rb

```diff
@@ -0,0 +1,190 @@
+
+# TODO: Move this to the release-toolkit
+#
+module Fastlane
+  module Helpers
+    class AndroidTools
+      attr_reader :android_sdk_root
+
+      def initialize(sdk_root: nil)
+        @android_sdk_root = sdk_root || ENV['ANDROID_HOME'] || ENV['ANDROID_SDK_ROOT'] || ENV['ANDROID_SDK']
+      end
+
+      def tool(paths:,binary:)
+        bin_path = `command -v #{binary}`.chomp
+        return bin_path unless bin_path.nil? || bin_path.empty? || !File.executable?(bin_path)
+
+        bin_path = paths
+          .map { |path| File.join(android_sdk_root, path, binary) }
+          .first { |path| File.executable?(path) }
+          
+        UI.user_error!("Unable to find path for #{binary} in #{paths.inspect}. Verify you installed the proper Android tools.") if bin_path.nil?
+        bin_path
+      end
+
+      def sdkmanager
+        @sdkmanager_bin ||= tool(paths: ['cmdline-tools', 'latest', 'bin'], binary: 'sdkmanager')
```

Comment:
TODO: I think depending on the installation and tool version and machine, sometimes it's called something else than `latest` (at least I vaguely remember having to deal with that difference between my local setup and how it was called on CI in a previous PR on another repo, even if I can't remember exactly where)